src/hotspot/cpu/aarch64/c2_CodeStubs_aarch64.cpp

```diff
@@ -75,6 +75,26 @@ void C2CheckLockStackStub::emit(C2_MacroAssembler& masm) {
   __ b(continuation());
 }
 
+int C2FixAnonOMOwnerStub::max_size() const {
+  return 20;
+}
+
+void C2FixAnonOMOwnerStub::emit(C2_MacroAssembler& masm) {
+  __ bind(entry());
+  Register mon = monitor();
+  Register t = tmp();
```

Comment:
assert that tmp is real, not noreg? Or is there a mechanism that already checks that?